pkg/odo/cli/storage/delete.go

```diff
@@ -77,6 +78,7 @@ func (o *StorageDeleteOptions) Run() (err error) {
 		}
 
 		log.Infof("Deleted storage %v from %v", o.storageName, o.localConfig.GetName())
+		log.Infof("Please use `odo push` command to delete the storage from the component")
```

Comment:
This might look a bit confusing maybe something like 
`Please use `odo push` command to delete the storage from the cluster`?

Reply:
Makes sense. Thanks for the simpler recommendation!